sdk/src/Core/Amazon.Runtime/CredentialManagement/CredentialProfileOptions.cs

```diff
@@ -111,6 +112,27 @@ public class CredentialProfileOptions
         /// </summary>
         public string WebIdentityTokenFile { get; set; }
 
+        /// <summary>
+        /// The AWS account ID that temporary AWS credentials will be resolved for using AWS SSO.
+        /// </summary>
+        public string SsoAccountId { get; set; }
```

Comment:
Do these need to be inside a target platform check? Similiar question down on lines 206/215.

Reply:
I opted for simpler complexity in this case, with the line of thought that these would be dormant fields otherwise -- the SDK would not identify the credential as an SSO profile type in those platforms.

Reply:
Got it, fine for me.

Reply:
The `EnsureCredentialProfileDetectorSetup` test in `sdk\test\Services\SecurityToken\UnitTests\Custom\StaticCheckTests.cs` says I'm wrong about this. I am adding more platform checks.